scraper.rb

```diff
@@ -51,6 +55,7 @@ def scrape_mp(page)
  }
  data[:party_id] = data[:party].gsub(/\W+/,'').downcase
  data[:image] &&= URI.join(page, data[:image].text.gsub(' ','%20s')).to_s
+ data[:phone] = multiple_values(data[:phone])
```

Comment:
It seems a little strange to do this as a separate step here, rather than in-place when the value is being extracted.